# **metaGAAP**

*metaGAAP v. 1.0* - **metaG**enome **A**ssembly and **A**nnotation **P**ipeline<br>
<br>

## **This piepeline is under development, please, be patient!**


### **To clone this repo**

```
git clone git@github.com:Gian77/metaGAAP.git
```

> **_NOTE_**<br> 
> This pipeline was born for running on the `HPCC` at Michigan State University which run the SLURM (Simple Linux Utility for Resource Management) job scheduler system. If you want to run this piepline in any other systems it will require substantial modification of the main, as well as, the accessory scripts.

## *This pipeline is based upon work supported by the Great Lakes Bioenergy Research Center, U.S. Department of Energy, Office of Science, Office of Biological and Environmental Research under award DE-SC0018409*

### **Software needed to fully run metaGAAP on HPCC**

Please install via conda (or use the binaries in the HPPC) of all these software below. 
* ##### [FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
* ##### [quast](http://bioinf.spbau.ru/quast)
* ##### [Kraken 2](https://ccb.jhu.edu/software/kraken2/)
* ##### [Bracken](https://github.com/jenniferlu717/Bracken)
* ##### [CheckM](https://ecogenomics.github.io/CheckM/)
* ##### [R](https://anaconda.org/r/r-essentials)


### **Acknowledgements**
Gian M.N. Benucci <benucci[at]msu[dot]edu> work was suppeorted by GLBRC (Great Lakes Bioenergy Research Center).

